### ISSUE TITLE
Add a11y audit on PF4 react integration environment

### ISSUE THREAD

**Issue by dlabaj** [patternfly/patternfly-react#1612]
In an effort to support a11y and ensure we catch a11y issues early on, we would like to run the a11y audit on the react components and fail the build if we exceed the teams recommended thresh hold.

Here's information on how to run it locally, we should look into getting this integrated somewhere in our build environment.

https://github.com/patternfly/patternfly-next#testing-for-accessibility

**Comment by redallen** [comment]
Closing in favor of #2336 .
